engine/table/src/main/java/io/deephaven/engine/table/impl/UncoalescedTable.java

```diff
@@ -65,6 +65,8 @@ public UncoalescedTable(@NotNull final TableDefinition definition, @NotNull fina
      */
     protected abstract Table doCoalesce();
 
+    protected void postCoalesceAction(final Table coalesced) {}
```

Comment:
Why isn't this part of `doCoalesce()`? You explained. I think you have access to all the stuff you care about via other means.